index.js

```diff
@@ -4,6 +4,7 @@ import getRawBody from 'raw-body'
 import assert from 'http-assert'
 import { validate } from './lib/validate.js'
 import { mapRequestToInetGroup } from './lib/inet-grouping.js'
+import { satisfies } from 'compare-versions'
```

Comment:
While [semver](https://www.npmjs.com/package/semver) is a popular package and it's maintained by the npm team, it has 93kB and depends on [lru-cache](https://www.npmjs.com/package/lru-cache). 

[compare-versions](https://www.npmjs.com/package/compare-versions) has zero dependencies and only 55kB.

Reply:
I'm happy to try the competitor module, but if it has bugs I would immediately first to replace it with `semver`, as bundle size doesn't matter much for our services imo